### ISSUE TITLE
Cherrypick tracking issue for v1.11

### ISSUE THREAD

**Issue by luxas** [kubernetes/kubeadm#958]
This issue serves as an umbrella to track the kubeadm-related PRs we cherrypick into the v1.11 branch.

I propose that @timothysc function as the cherrypick lead on the kubeadm side

I've identified a list of PRs to be cherrypicked:

 - [x] Support specifying kube-proxy tolerations: https://github.com/kubernetes/kubeadm/issues/699
   - PR to master: https://github.com/kubernetes/kubernetes/pull/65931
   - Cherrypick PR: https://github.com/kubernetes/kubernetes/pull/66043
   - Assignee: @neolit123 
 - [x] Getting extra output when running kubeadm token create command: https://github.com/kubernetes/kubeadm/issues/953
   - PR to master: https://github.com/kubernetes/kubernetes/pull/65584
   - Cherrypick PR: https://github.com/kubernetes/kubernetes/pull/65652
   - Assignee: @neolit123 
 - [x] CoreDNS: wrong port for Prometheus in 1.11: https://github.com/kubernetes/kubeadm/issues/955
   - PR to master: https://github.com/kubernetes/kubernetes/pull/65589
   - Cherrypick PR: https://github.com/kubernetes/kubernetes/pull/65650
   - Assignee: @rajansandeep 
 - [x] Bazel bugfixes to match k8s/release
   - [ ] Adds cri-tools as a dependency to kubeadm deb/rpms https://github.com/kubernetes/kubernetes/pull/65313
      - 1st Cherrypick PR: https://github.com/kubernetes/kubernetes/pull/65921
      - 1st Revert PR: https://github.com/kubernetes/kubernetes/pull/66259
      - 2nd Cherrypick PR: unknown
      - Assignee: @rdodev 
   - [x] bazel: kubelet deb package postinstall step https://github.com/kubernetes/kubernetes/pull/65554
      - Cherrypick PR: https://github.com/kubernetes/kubernetes/pull/65922
      - Assignee: @rdodev 
 - [x] kubeadm tries to pull `k8s.gcr.io/pause-amd64:3.1` instead of `k8s.gcr.io/pause:3.1` https://github.com/kubernetes/kubeadm/issues/962
   - PR to master: https://github.com/kubernetes/kubernetes/pull/65920
   - Cherrypick PR: https://github.com/kubernetes/kubernetes/pull/65923
   - Assignee: @dims 
 - [x] kubeadm: stop setting UID in the kubelet ConfigMap https://github.com/kubernetes/kubeadm/issues/921
   - PR to master: https://github.com/kubernetes/kubernetes/pull/66341
   - Cherrypick PR: https://github.com/kubernetes/kubernetes/pull/66495
   - Assignee: @runiq
 - [x] kubeadm: Pull sidecar and dnsmasq-nanny images when using kube-dns
   -  PR to master: https://github.com/kubernetes/kubernetes/pull/66499
   - Cherrypick PR: https://github.com/kubernetes/kubernetes/pull/66679
   - Assignee: @neolit123

Individual cherrypick PRs should be created as documented here: https://github.com/kubernetes/community/blob/master/contributors/devel/cherry-picks.md

TL;DR; download https://github.com/github/hub, make sure `git remote -v` looks something like this and execute:

```bash
$ git remote -v
origin	git@github.com:luxas/kubernetes.git (fetch)
origin	git@github.com:luxas/kubernetes.git (push)
upstream	https://github.com/kubernetes/kubernetes (fetch)
upstream	DISABLE (push)

# Example command
GITHUB_USER=luxas hack/cherry_pick_pull.sh upstream/release-1.11 12345
```

The script will do everything for you, including creating the PR. You only need to get approval from @timothysc / me as kubeadm approvers and @foxish that is [the v1.11 patch manager](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.11/release_team.md) and then the cherrypick can be merged!

**Comment by neolit123** [comment]
closed as per slack discussion.
